packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3927,6 +3930,7 @@ class IndexedStack extends Stack {
     assert(_debugCheckHasDirectionality(context));
     renderObject
       ..index = index
+      ..fit = fit
```

Comment:
`updateRenderObject` also needs to update the clipBehavior on the RenderObject.

Reply:
Thanks!  Fixed.